src/test/article.spec.js

```diff
@@ -213,3 +219,246 @@ describe('GET api/v1/articles', () => {
+          image: articleSample.image
+        });
+      expect(response.status).to.equal(200);
+      expect(response.body.message).to.equal('Article successfully updated');
+      expect(response.body.data).to.have.property('id');
+      expect(response.body.data.id).to.be.a('number');
+      expect(response.body.data).to.have.property('title');
+      expect(response.body.data.title).to.equal(articleSample.title);
+      expect(response.body.data).to.have.property('description');
+      expect(response.body.data.description).to.equal(
+        articleSample.description
+      );
+      expect(response.body.data).to.have.property('body');
+      expect(response.body.data.body).to.equal(articleSample.body);
+      expect(response.body.data).to.have.property('image');
+      expect(response.body.data.image).to.equal(articleSample.image);
+      expect(response.body.data).to.have.property('slug');
+      expect(response.body.data.slug).to.equal(articleSample.slug);
+      expect(response.body.data).to.have.property('authorId');
+      expect(response.body.data).to.have.property('publishedDate');
+      expect(response.body.data).to.have.property('createdAt');
+      expect(response.body.data).to.have.property('updatedAt');
+    });
+
+    it('should throw a 404 error when a specific article is not found', async () => {
+      const validArticle = await BaseRepository.create(
+        db.Article,
+        articleSample
+      );
+      const response = await chai
+        .request(app)
+        .get(`${ARTICLES_API}/000010000100`)
+        .set('x-access-token', token)
+        .send(!validArticle);
+      expect(response.status).to.equal(404);
+      expect(response.body.message).to.equal(
+        'The requested article was not found'
+      );
+    });
+  });
+
+  describe('DELETE /api/v1/articles/:articleId', () => {
+    beforeEach(async () => {
+      await db.Bookmark.destroy({ cascade: true, truncate: true });
+      await db.User.destroy({ cascade: true, truncate: true });
+      await db.Article.destroy({ cascade: true, truncate: true });
+    });
+    it('should successfully delete an article', async () => {
+      const validArticle = await BaseRepository.create(
+        db.Article,
+        articleSample
+      );
+      const response = await chai
+        .request(app)
+        .delete(`${ARTICLES_API}/${validArticle.id}`)
+        .set('x-access-token', token);
+      expect(response.status).to.equal(200);
+      expect(response.body.message).to.equal('Article successfully deleted');
+    });
+
+    it('should throw a 404 error when a specific article is not found', async () => {
+      const validArticle = await BaseRepository.create(
+        db.Article,
+        articleSample
+      );
+      const response = await chai
+        .request(app)
+        .get(`${ARTICLES_API}/000010000100`)
+        .set('x-access-token', token)
+        .send(!validArticle);
+      expect(response.status).to.equal(404);
```

Comment:
This should be a success. If a resource doesn't exist at the time of a delete request, then it's a success because the point wasn't to check where it exists. Instead, the point is to ensure it doesn't exist